wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -587,15 +666,15 @@ <h3>Image transparency</h3>
 
     {% include "wagtailadmin/pages/_editor_js.html" %}
 
-    {{ example_form.media.js}}
+    {{ example_form.media.js }}
 
     <script>
-        $(function(){
-            (function runprogress(){
-                var to = setTimeout(function(){
+        $(function () {
+            (function runprogress() {
+                var to = setTimeout(function () {
                     runprogress();
                     clearTimeout(to);
-                    setTimeout(function(){
+                    setTimeout(function () {
```

Comment:
I'd recommend against formatting this JS for now, it's a bit of a distraction on this PR. 

We may look at a seperate issue to pull this JS out to an external script in the future anyway.